doc/qbittorrent.1.md

```diff
@@ -3,34 +3,165 @@
 % January 16th 2010
 
 # NAME
-qBittorrent - a Bittorrent client written in C++ / Qt
 
+**qBittorrent** - a BitTorrent client written in C++ / Qt
 
 # SYNOPSIS
-**qbittorrent** `[--no-splash] [--webui-port=x] [TORRENT_FILE | URL]...`
-
-**qbittorrent** `--help`
-
-**qbittorrent** `--version`
 
+| **qbittorrent** \[_OPTIONS_] \[_TORRENT_FILE_ | _URL_ ...]
+| **qbittorrent** **-h | -v**
 
 # DESCRIPTION
+
 **qBittorrent** is an advanced Bittorrent client written in C++ / Qt,
-using the **libtorrent-rasterbar** library by Arvid Norberg. qBittorrent is similar to uTorrent.
-qBittorrent is fast, stable, light, it supports unicode and it provides a good integrated
-search engine. It also comes with UPnP port forwarding / NAT-PMP, encryption (Vuze compatible),
-FAST extension (mainline) and PeX support (utorrent compatible).
+using the **libtorrent-rasterbar** library by Arvid Norberg as the backend. qBittorrent provides a familiar GUI similar to uTorrent's, and is fast, stable and light.
+
+qBittorrent supports several important modern features such as UPnP port forwarding / NAT-PMP, kademelia DHT (Mainline) and magnet link support, IPv6, uTP, PeX, LSD, UDP tracker protocol support and more. Refer to the **libtorrent-rasterbar** documentation for the full list of features and supported BEPs.
+
+In addition, qBittorrent supports Unicode and it provides a good integrated Python-based torrent search engine, an RSS downloader, a torrent creation GUI, a basic embedded tracker to facilitate sharing your own torrents with friends, and an HTTP API/Web UI combo for remote access/administration.
+
+If no arguments are specified, qBittorrent simply starts and shows the GUI.
```

Comment:
Probably worth mentioning that stdout and stderr will be attached to the console if starting from a console.

Reply:
Fixed.